class LogoutError(Exception):
    pass

class LoginError(Exception):
    pass